catroid/src/androidTest/java/org/catrobat/catroid/test/utils/TestUtils.java

```diff
@@ -92,4 +97,76 @@ public static Pixmap createRectanglePixmap(int width, int height, Color color) {
 		pixmap.fillRectangle(0, 0, width, height);
 		return pixmap;
 	}
+
+	public static Project createProjectWithGlobalValues(String name, String spriteName, String valueName, Context context) {
```

Comment:
please dont mix nomenclatures, variables != values

Reply:
I changed it to createProjectWithGlobalVariables